pandas-stubs/core/series.pyi

```diff
@@ -1548,11 +1548,13 @@ class Series(IndexOpsMixin[S1], NDFrame):
     ) -> Series[S1]: ...
     def pct_change(
         self,
-        periods: int = ...,
-        fill_method: _str = ...,
+        periods: int = 1,
+        fill_method: None = ...,
         limit: int | None = ...,
-        freq=...,
-        **kwargs: Any,  # TODO: make more precise https://github.com/pandas-dev/pandas-stubs/issues/1169
+        freq: Frequency | timedelta | None = None,
+        *,
+        axis: AxisIndex = ...,
+        fill_value: object | None = ...,
```

Comment:
Same comment as above on `DataFrame.pct_change()` .  Also, remove `axis` as it is not used in `Series.pct_change()`.